Procfile

```diff
@@ -1 +1 @@
-web: node index.js
```

Comment:
Do we need this on heroku or was it just for development?

Reply:
We do need it on heroku
I had to change so that pm2 runs the start script